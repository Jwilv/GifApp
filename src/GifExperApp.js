import React, { useState } from "react";
import { AddCategory } from "./components/AddCategory";
import { GifGrid } from "./components/GifGrid";

export const GifExperApp = () => {
  const [categories, setCategories] = useState([]);

  return (
    <>
      <h2>GIFS APP</h2>
      <AddCategory setCategories={setCategories} />
      <hr /> 
          <ol>
            {
            categories.map(category => (
              <GifGrid
                key={category}
                category={category}
              />
            ))
            }
          </ol>
    </>
  );
};
