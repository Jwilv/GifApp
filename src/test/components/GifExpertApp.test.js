import React from 'react'
import { shallow } from 'enzyme'
import { GifExperApp } from '../../GifExperApp';

describe('pruebas en <GifExpertApp />', () => {
    const wrapper = shallow(<GifExperApp />)
    test('debe renderizarse correctamente', () => {
        
    expect(wrapper).toMatchSnapshot();
        
    });
});