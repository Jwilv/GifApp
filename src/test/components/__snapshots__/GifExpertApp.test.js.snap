// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <GifExpertApp /> debe renderizarse correctamente 1`] = `
<Fragment>
  <h2>
    GIFS APP
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</Fragment>
`;
